modules/mono/editor/GodotTools/GodotTools/Build/BuildSystem.cs

```diff
@@ -0,0 +1,174 @@
+using GodotTools.Core;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using GodotTools.BuildLogger;
+using GodotTools.Internals;
+using GodotTools.Utils;
+using Directory = System.IO.Directory;
+using Environment = System.Environment;
```

Comment:
Aren't these already available because of `using System;` and `using System.IO;`?

Reply:
Hmm, when I do that in other files it's because of collision with the `Godot` namespace; but that's not imported here.

Reply:
Kept the Directory one as it conflicts with `GodotTools.Utils.Directory`.